### DISCUSSION TITLE
Names and emails on GitHub

### DISCUSSION THREAD

**Discussion by Falesz** [Buduf/PTI-Ear-Trainer#10] [Q&A]
Hey all,
I'm not sure how GitHub reads our names and email addresses.
I have successfully set my name to "Lámfalusi Ádám" and my email to my Uni email address (I tested it), but in the repo commits and the history I can only see our GitHub usernames.
Where can I see what name and email a collaborator used for a commit in git?
Shouldn't I be seeing your full names in the repo as well? (Currently I can only see the GitHub usernames for everybody)

**Comment by Buduf** [comment]
I don't think you can view it on GitHub (because of security reasons). But if you run `git log` command it will list the commits with the details.

**Reply by Buduf** [reply]
@Andre2220 's commits don't show the correct email and name, although we set it with `git config`. I believe thats because he commited with Visual Studio and you have to set it separately (in menu Git -> Settings). Also there's a commit with a ...@github.com email, because we merged the branches on GitHub in a Pull Request. So we might have to do that on our computer next time.

**Reply by Falesz** [reply]
I just checked my settings in Visual Studio and the correct parameters are set. GitHub just doesn't utilize that information, instead it reads our GitHub usernames.

**Comment by Falesz** [comment]
The command 'git log' does indeed show the correct names and email addresses, so I hope that will do for the teachers.

**Comment by Andre2220** [comment]
The command 'git log' shows my github username and email for my commits on the 'game-logics' branch but I've already set the Uni email and my full name for locally this branch, so I'm not sure what could cause the problem. Do you have any idea?

**Reply by Buduf** [reply]
Did you try to set it in Visual Studio?
